app/assets/javascripts/admin/controllers/admin-badges.js.es6

```diff
@@ -1,2 +1,19 @@
 import Controller from "@ember/controller";
-export default Controller.extend();
+import { inject as service } from "@ember/service";
+
+export default Controller.extend({
+  selectedRoute: "adminBadges.show",
+  router: service(),
+
+  actions: {
+    showBadge() {
```

Comment:
Could we convert these actions to links? Buttons can have `href`s

Reply:
I replaced the transition with an "href". However, I still need the action to dynamically set the badges link-to url.